plaid/institutions.go

```diff
@@ -1,19 +1,149 @@
 package plaid
 
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+)
+
 // GetInstitution returns information for a single institution given an ID.
-// See https://plaid.com/docs/api/#institutions-by-id.
+// See: https://plaid.com/docs/api/#institutions-by-id
 func GetInstitution(environment environmentURL, id string) (inst institution, err error) {
-	err = getAndUnmarshal(environment, "/institutions/"+id, &inst)
+	err = getAndUnmarshal(environment, "/institutions/all/"+id, &inst)
```

Comment:
Let's check `id` is not empty. Otherwise we get a strange error trying to hit a non-existent `GET /institutions/all/` (without param) route.